code/game/objects/items/melee/swords.dm

```diff
@@ -0,0 +1,298 @@
+
+/obj/item/melee/sword/rust/cultblade
+	name = "rusty dark blade"
+	desc = "Once used by worshipers of forbidden gods, now its covered in old rust."
+	icon_state = "cultblade_rust"
+	inhand_icon_state = "cultblade_rust"
+	broken_icon = "cultblade_broken"
+
+/obj/item/melee/sword/claymore
+	name = "holy claymore"
+	desc = "A weapon fit for a crusade! It lacks a holy shine however."
+	force = 18
+	icon_state = "claymore_gold"
+	inhand_icon_state = "claymore_gold"
+	worn_icon_state = "claymore_gold"
+
+/obj/item/melee/sword/claymore/darkblade
+	name = "dark blade"
+	desc = "Spread the glory of the dark gods! Even if they don't bless this blade."
+	icon_state = "cultblade"
+	inhand_icon_state = "cultblade"
+	worn_icon_state = "cultblade"
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+
+/obj/item/melee/sword/reforged
+	name = "Reforged longsword"
+	desc = "A hard steel blade, it's edge has been forged to be incredibly strong. It feels light."
+	icon_state = "reforged"
+	inhand_icon_state = "reforged"
+	worn_icon_state = "reforged"
+	force = 25
+	throwforce = 15
+	throw_range = 6
+	block_chance = 40
+	wound_bonus = 5
+	armour_penetration = 15
+	embedding = list("embed_chance" = 30, "impact_pain_mult" = 10)
+
+/obj/item/melee/sword/reforged/shitty
+	var/broken = FALSE
+	var/rustiness = 0 //This isn't a mistake, this causes it to break instantly upon use.
+	var/broken_icon = "reforged_broken"
+
+/obj/item/melee/sword/reforged/shitty/afterattack(target, mob/user, proximity_flag)
+	. = ..()
+	if(broken)
+		return ..()
+	if(ismovable(target))
+		decrease_uses(user)
+
+/obj/item/melee/sword/reforged/shitty/hit_reaction(mob/user)
+	. = ..()
+	if(broken)
+		return ..()
+	if(!.)
+		return
+	decrease_uses(user)
+
+/obj/item/melee/sword/reforged/shitty/proc/decrease_uses(mob/user)
+	if(rustiness == 0)
+		no_uses(user)
+		return
+	rustiness--
+
+/obj/item/melee/sword/reforged/shitty/proc/no_uses(mob/user)
+	if(broken == TRUE)
+		return
+	user.visible_message(span_notice("[user]'s sword breaks. WHAT AN IDIOT!"), span_notice("The [src]'s blade shatters! It was a cheap felinid imitation! WHAT A PIECE OF SHIT!"))
+	broken = TRUE
+	name = "broken fake longsword"
+	desc = "A cheap piece of felinid forged trash."
+	icon_state = broken_icon
+	inhand_icon_state = broken_icon
+	worn_icon_state = broken_icon
+	update_appearance()
+	playsound(user, 'sound/effects/glassbr1.ogg', 100, TRUE)
+	force -= 20
+	throwforce = 5
+	throw_range = 1
+	block_chance = 5
+	wound_bonus = -10
+	armour_penetration = 0
+	embedding = list("embed_chance" = 5, "impact_pain_mult" = 5)
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/storage/belt/sheath
+	name = "sword sheath"
+	desc = "A leather sheath meant to hold a variety of swords."
+	icon_state = "sheath_plain"
+	inhand_icon_state = "sheath_plain"
+	worn_icon_state = "sheath_plain"
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
I almost wonder if the captain's sabre sheathe should inherent from this, since this also copies code from that.
IE, `/obj/item/storage/belt/sheath/sabre`.

Reply:
You can just override the lists to ensure that doesn't happen.